nexussdk/identities.py

```diff
@@ -6,24 +6,27 @@
 
 from typing import Dict
 
-from nexussdk.utils.http import http_get
+from nexussdk.utils.http import Http
 
-SEGMENT = "identities"
 
+class Identities:
+    segment = "identities"
 
-# Read functions.
+    def __init__(self, http: Http):
+        self._http = http
 
-def fetch() -> Dict:
-    """Fetch the identities.
+    # Read functions.
 
-    :return: A list with the Nexus payloads of the identities.
-    """
-    return http_get([SEGMENT])
+    def fetch(self) -> Dict:
+        """Fetch the identities.
 
+        :return: A list with the Nexus payloads of the identities.
+        """
+        return self._http.get([Identities.segment])
```

Comment:
You can access class attributes (`segment`) with `self`. It would make more explicit that this is something from this class and not an other.